source/iface/platform.rst

```diff
@@ -36,7 +38,8 @@ A platform contains 1 or more SYCL devices, or a host device.
 
```

Comment:
Need to update constructor code samples

Reply:
Added issue and waiting response : https://github.com/KhronosGroup/SYCL_Reference/issues/67